test/node/conference-bridge.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let user;
+let client2;
+let user2;
+let conference;
+describe('Conference', () => {
+
+    before(async () => {
+        client = new Circuit.Client(config.bot1);
+        user = await client.logon();
+        client2 = new Circuit.Client(config.bot2);
+        user2 = await client2.logon();
+    });
+
+    after(async () => {
+        await client.logout();
+        await client2.logout();
+    });
+
+
+    it('should create a conference bridge', async () => {
+        const topic = `${Date.now()}a`;
+        const res = await Promise.all([
+            client.createConferenceBridge(topic),
+            helper.expectEvents(client, [{
+                type: 'conversationCreated',
+                predicate: evt => evt.conversation.topic === topic && evt.conversation.creatorId === user.userId
+            }])
+        ]);
+        conference = res[0];
```

Comment:
Shouldn't you check if a conference is returned using assert?

Reply:
done